.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.watcherExclude": {
+        "**/target": true
+    }
+}
```

Comment:
That file is IDE specific. I would remove it from git. Feel free to add `.vscode` in the `.gitignore` if you want.